learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+  -H 'Authorization: Bearer Meilisearch_API_Key'
+  --data-binary '{ 
+    "eventType": "conversion",
+    "eventName": "product added to cart",
+    "indexUid": "products",
+    "objectID": "0",
+    "position": 0
+  }'
+```
+
+It is not possible to associate multiple `conversion` events with the same search.
+
+#### The `conversion` event object
+
+The `conversion` event must deliver an object with the following fields:
+
+```js
+{
+  "eventType": "conversion",
+  "eventName": "product added to cart",
+  "indexUid": "products",
+  "objectID": "0",
+  "position": 0
+}
+```
+
+- `eventType`: indicates this is a `conversion` event
+- `eventName`: an arbitrary string describing the event
+- `indexUid`: the document's index
+- `objectID`: the document's primary key
+- `position`: the document's position in the list of search results
+
+## Understanding the analytics interface
+
+### Total users and user ids
+
+Indicates the total number of user ids. User ids are set with the `X-MS-USER-ID` HTTP header. If `X-MS-USER-ID` is not present, user ids are generated based on `User-Agent` and `CF-Connecting-IP`.
+
+### Total searches and search IDs
+
+The total number of generated search ids. Meilisearch Cloud generates a new search id for each [user id](#total-users-and-user-ids) after 3 seconds of user inactivity. Multiple queries may be associated with a single search id.
+
+### Most searched queries 
+
+Indicates the most common search terms in the specified time period. Meilisearch Cloud calculates this based on [search ids](#total-searches-and-search-ids). If multiple queries correspond to the same search id, Meilisearch Cloud only tracks the last query associated with it.
+
+### Searches with no results and no result rate
+
+Searches with no results indicates search terms that did not return any results. If multiple queries correspond to the same [search id](#total-searches-and-search-ids), Meilisearch Cloud only tracks the last query associated with it.
+
+The no result rate is the number of searches with no results divided by the total number of searches.
+
+### Conversion rate
+
+A conversion is when users take action relevant to your business beyond clicking on a search result. Exactly what counts as a conversion depends on your specific application. For example, a web shop might count adding a product to their shopping cart as a conversion.
+
+Conversion rate indicates the number of searches that resulted in a `conversion` event divided by the total number of searches.
+
+To track conversion rates, [configure your application to send `conversion` events to Meilisearch Cloud](#configuring-conversion-rate).
+
+### Click-through rate and average click position
+
+Click-through rate is the number of times users have clicked on a search result divided by the total number of searches.
+
+Average click position indicates the average position of the search results users clicked on, with 0 indicating the first search result. 
```

Comment:
Is it `0` or `1` for the first position? Cross-linking @macraig message [here](https://meilisearch.slack.com/archives/C060MLU0X44/p1700050365021069?thread_ts=1700048907.819179&cid=C060MLU0X44)

Reply:
We changed the lowest value to 1 in [this pr](https://github.com/meilisearch/meilisearch-cloud/pull/3852) but as of Nov 16 12PM CET it's not deployed yet